deployment/gke/deployment_manager_configs/cluster.jinja

```diff
@@ -159,8 +160,8 @@ resources:
         # Set min cpu platform to ensure AVX2 is supported.
         minCpuPlatform: 'Intel Broadwell'
         accelerators:
-          - acceleratorCount: 1
-            acceleratorType: {{ properties['gpu-type'] }}
+          - acceleratorCount: {{ properties['gpu-number-per-node'] }}
+            acceleratorType: nvidia-tesla-k80
```

Comment:
Why did you remove gpu-type? Users will likely want to configure this.

K80s are pretty old at this point.

Reply:
Updated